### PR TITLE
Change the function & attribute names Sanitizer interface and SanitizerConfig dictionary

### PR DESCRIPTION
**Pull request by iVanlIsh**
This PR intend to:
1. Change `toString` to `sanitizeToString`.
2. Change `toFragment` to `sanitize`.
3. Add `CreationOptions` attribute into `Sanitizer` interface.
4. Change the `SanitizerConfig`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,17 +88,19 @@ We want to develop an API that learns from the
...
-interface Sanitizer {
-  constructor(SanitizerConfig? config);
-  DOMString toString(DOMString input);
-  DocumentFragment toFragment(DOMString input);
-
-  readonly SanitizerConfig creation_options;
+[
+  Exposed=Window
+] interface Sanitizer {
+  [RaisesException] constructor(SanitizerConfig? config);
```
Comment (otherdaniel) [inline README.md:99]: Please remove [RaisesException].

RaisesException is a Blink-specific attribute, telling our code generator to add ExceptionState-references to the callback signature. This doesn't seem to have an equivalent in the standards world.

(Reference: https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/bindings/IDLExtendedAttributes.md#RaisesException_i_m_a)

**Review by otherdaniel — APPROVED** [review]
Looks good, but let's keep freddyb in the loop, because IIRC this partially overlaps with one of his changes.

File: README.md (hunk 1 above)
Comment (mikewest) [reply README.md:99]: I recall @yuki3 suggesting we make this optional (e.g. `optional SanitizerConfig config = {}`) rather than nullable (via `?`).

Comment (iVanlIsh) [reply README.md]: You are right and done.

File: README.md [hunk 2]
```diff
@@ -88,17 +88,19 @@ We want to develop an API that learns from the
...
 
 ```
-interface Sanitizer {
-  constructor(SanitizerConfig? config);
-  DOMString toString(DOMString input);
-  DocumentFragment toFragment(DOMString input);
-
-  readonly SanitizerConfig creation_options;
+[
+  Exposed=Window
```
Comment (mozfreddyb) [inline README.md:97]: `SecureContext`, please

**Comment by mozfreddyb** [comment]
please excuse the delay. looks good!

**Review by mozfreddyb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (iVanlIsh) [reply README.md]: Done.

**Pull request merged**